e2e/test/scenarios/dashboard-filters/dashboard-filters-source.cy.spec.js

```diff
@@ -286,11 +340,17 @@ const mapFilterToQuestion = () => {
   popover().within(() => cy.findByText("Category").click());
 };
 
-const filterDashboard = ({ isField = false, isSandboxed = false } = {}) => {
+const filterDashboard = ({
+  isField = false,
+  isSandboxed = false,
+  isLabeled = false,
+} = {}) => {
   cy.findByText("Text").click();
 
   popover().within(() => {
-    cy.findByText("Gizmo").should("be.visible");
+    const Gizmo = isLabeled ? "Gizmo Label" : "Gizmo";
```

Comment:
nit: we use camelCase or UPPER_CASE for these
```suggestion
    const gizmo = isLabeled ? "Gizmo Label" : "Gizmo";
```